examples/complete/variables.tf

```diff
@@ -99,3 +99,13 @@ variable "topics" {
     }
   ]
 }
+
+variable "service_credential_names" {
+  description = "Map of name, role for service credentials that you want to create for the event streams"
+  type        = map(string)
+  default = {
+    "es_writer" : "Writer",
+    "es_reader" : "Reader",
+    "es_manager" : "Manager"
```

Comment:
how about testing the `None` value too? Also can you add this directly to the example main.tf instead of exposing it as a variable?

Reply:
I got an error while testing locally that "None" is not a valid role.

Reply:
@dishankkalra23 you still have not added the `service_credential_names` vale directly in main.tf.